test/auth.e2e-spec.ts

```diff
@@ -0,0 +1,159 @@
+import { MailerModule } from '@nestjs-modules/mailer';
+import { BullModule } from '@nestjs/bullmq';
+import { INestApplication } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { APP_GUARD } from '@nestjs/core';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Sequelize } from 'sequelize-typescript';
+import { Entities } from 'src/app.module';
+import { AuthModule } from 'src/auth/auth.module';
+import { SingInAuthUserDto } from 'src/auth/dto/create/create-auth.dto';
+import { User } from 'src/auth/entities/user.entity';
+import { RegistrationMethod } from 'src/auth/enums/registMethod-enum';
+import { SendCodeService } from 'src/auth/services/sendCode.service';
+import { AccessTokenGuard } from 'src/common/guards/jwt/accessToken.guard';
+import { AccessTokenStrategy } from 'src/common/strategies/accessToken.strategy';
+import { RefreshTokenStrategy } from 'src/common/strategies/refreshToken.strategy';
+import * as request from 'supertest';
+
+describe('Auth', () => {
+  let app: INestApplication;
+  let sequelize: Sequelize;
+  let createdUser: User;
+  let sendCodeService: SendCodeService;
+  let accessToken: string;
+  let refreshToken: string;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [
```

Comment:
In this case you can just import the whole `app.module` like in the following example - https://dev.to/grocstock/nestjs-unit-and-e2e-testing-7pb